src/java/org/apache/cassandra/index/sai/utils/RangeIterator.java

```diff
@@ -25,6 +25,7 @@
 
 import com.google.common.annotations.VisibleForTesting;
 
+import org.apache.cassandra.index.sasi.disk.Token;
```

Comment:
Nit: unused import.

Reply:
Interesting that sonar did not catch that one.

Reply:
Sonar must not be set up to catch them for this configuration because I have never seen it catch it. I only commented on it because others have asked me to remove them in my PRs. It's definitely something that is better when automated.